Signum.React/Scripts/SearchControl/SearchModal.tsx

```diff
@@ -106,6 +106,7 @@ export default class SearchModal extends React.Component<SearchModalProps, { sho
             hideFullScreenButton={true}
             throwIfNotFindable={true}
             ref={e => this.searchControl = e!}
+            create={this.props.findMode == "Explore"}

```

Comment:
This overrides the EntitySettings / QuerySettings configuration in every case. Let's discuss this together next meeting.

Reply:
Ok